tests/Aggregation/DateHistogramTest.php

```diff
@@ -99,7 +99,7 @@ public function testSetOffset(): void
         $expected = [
             'date_histogram' => [
                 'field' => 'created',
-                'interval' => '1h',
```

Comment:
Do we have test covering the fixed_interval alternative?

Reply:
I added two different test of the construct method